secator/output_types/domain.py

```
@@ -0,0 +1,52 @@
+import time
+from datetime import datetime
+from dataclasses import dataclass, field
+from enum import Enum
```

Ruff (0.11.9)

`enum.Enum` imported but unused

Remove unused import: `enum.Enum`

(F401)